http.c

```diff
@@ -4036,11 +4094,17 @@ evhttp_set_default_content_type(struct evhttp *http,
 }
 
 void
-evhttp_set_allowed_methods(struct evhttp* http, ev_uint16_t methods)
+evhttp_set_allowed_methods(struct evhttp* http, ev_uint32_t methods)
```

Comment:
This requires separate patch

Reply:
isn't 0757534f6e6eed1e37588ada9a38c661999651dc "separate" enough ?
I think you can cherry-pick it in master